components/google-cloud/google_cloud_pipeline_components/experimental/custom_job/custom_job_sample.ipynb

```diff
@@ -0,0 +1,348 @@
+   "source": [
+    "PROJECT_ID = \"python-docs-samples-tests\"  # @param {type:\"string\"}\n",
+    "BUCKET_NAME = \"gs://[your-bucket-name]\"  # @param {type:\"string\"}\n",
+    "REGION = \"us-central1\"  # @param {type:\"string\"}\n",
+    "PIPELINE_ROOT = \"{}/pipeline_root\".format(BUCKET_NAME)\n",
+    "PIPELINE_ROOT"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Import libraries"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "import google_cloud_pipeline_components as gcpc\n",
+    "from kfp.v2 import components, compiler, dsl"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Create a component, convert it to a custom job to use in a pipeline.\n",
+    "Create a simple component that takes an input and produces an output"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "producer_op = components.load_component_from_text(\n",
+    "    \"\"\"\n",
+    "name: Producer\n",
+    "inputs:\n",
+    "- {name: input_text, type: String, description: 'Represents an input parameter.'}\n",
+    "outputs:\n",
+    "- {name: output_value, type: String, description: 'Represents an output paramter.'}\n",
+    "implementation:\n",
+    "  container:\n",
+    "    image: google/cloud-sdk:latest\n",
+    "    command:\n",
+    "    - sh\n",
+    "    - -c\n",
+    "    - |\n",
+    "      set -e -x\n",
+    "      echo \"$0, this is an output parameter\" | gsutil cp - \"$1\"\n",
+    "    - {inputValue: input_text}\n",
+    "    - {outputPath: output_value}\n",
+    "\"\"\"\n",
+    ")"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Convert the component to a Vertex AI Custom Job"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "# This is a temporary work around for v0.1.4 to point to the correct base image for custom_job\n",
+    "gcpc.experimental.custom_job.custom_job._DEFAULT_CUSTOM_JOB_CONTAINER_IMAGE = 'gcr.io/ml-pipeline/gcp-launcher:0.1.4'\n",
+    "\n",
+    "custom_job_producer_op= gcpc.experimental.custom_job.custom_job.run_as_vertex_ai_custom_job(producer_op)"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Define the pipeline:"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "@dsl.pipeline(pipeline_root='', name='custom-job-sample-pipeline')\n",
+    "def pipeline(text: str = 'message'):\n",
+    "    custom_producer_task = custom_job_producer_op(input_text=text, gcp_project=PROJECT_ID ,gcp_region=REGION)"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Compile and run the pipeline\n",
```

Comment:
same here. i think this can be moved to a common notebook for all FPC

Reply:
done.